### PR TITLE
add basic issue template

### PR DESCRIPTION
**Pull request by jlipps**
cc @imurchie @bootstraponline

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## The problem
+
+This should be a brief description of the issue you are experiencing (or the feature you want to see added to Appium). Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to http://discuss.appium.io!
```
Comment (bootstraponline) [inline issue_template.md]: > This should be a brief description of the issue you are experiencing (or the feature you want to see added to Appium).

I think this reads better as:

> Briefly describe the issue you are experiencing (or the feature you want to see added to Appium).

File: issue_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Environment
+
+* Appium version (or git revision) that exhibits the issue:
+* Last Appium version that did not exhibit the issue (if applicable):
+* System and version:
```
Comment (bootstraponline) [inline issue_template.md]: What does system refer to? The desktop operating system? The mobile system?

File: issue_template.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+* Mobile platform/version under test:
+* Real device or emulator/simulator:
+* Appium CLI or Appium.app|exe:
+
+## Details
+
+This is the place where you can go into more detail about the problem you have been experiencing, if necessary. Otherwise, feel free to delete it.
```
Comment (bootstraponline) [inline issue_template.md]: I think this reads cleaner as:

> Describe in more detail the problem you have been experiencing, if necessary.

File: issue_template.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+## Link to Appium logs
+
+Create a [gist](https://gist.github.com) which is a paste of your _full_ Appium logs, and link them here. Do _not_ paste your full Appium logs here, as it will make this issue very long and hard to read! If you are reporting a bug, _always_ include Appium logs!
```
Comment (bootstraponline) [inline issue_template.md]: I will miss the 50 page issues full of nothing but debug logs :)

**Comment by jlipps** [comment]
updated per @bootstraponline's comments

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+## The problem
+
+Briefly describe the issue you are experiencing (or the feature you want to see added to Appium). Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to http://discuss.appium.io!
+
+## Environment
+
+* Appium version (or git revision) that exhibits the issue:
+* Last Appium version that did not exhibit the issue (if applicable):
+* Desktop system and version used to run Appium:
```
Comment (bootstraponline) [inline .github/ISSUE_TEMPLATE.md]: `Desktop operating system` since we don't care about the hardware.

**Comment by bootstraponline** [comment]
Looks great! I think this will have a positive impact on the quality of new issues.

**Comment by imurchie** [comment]
Looks good to me! This is a pretty cool new feature!

**Comment by jlipps** [comment]
cool! not waiting for travis since this is a markdown file. thanks!

**Comment by jlipps** [comment]
also merging into master so we can see it in action right away.

**Pull request merged**